mu4/importexport/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -2362,6 +2362,7 @@ void MusicXMLParserPass2::staffTuning(StringData* t)
 
 void MusicXMLParserPass2::measureStyle(const QString& partId, Measure* measure)
 {
+    Q_UNUSED(partId);
```

Comment:
Please, remove this argument from the method signature

Reply:
That's the other solution, yes. I though that maybe @lvinken has further plans with this, so left it in for now.

Reply:
Also, you should use our UNUSED macro (defined in log.h) in mu4. And please, don't add ; because macro already contains ; inside it and this can be a reason for a new warning :)

Reply:
Well, QtCreator warns about the missing semicolon IIRC

Reply:
I'm OK with removing the argument. As of now, I have no recollection of any plan to use it and if I need it in future I'll simply add it back myself.